app/models/planet.py

```diff
@@ -0,0 +1,22 @@
+from app import db
+
+
+class Planet(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String)
+    description = db.Column(db.String)
+    distance_from_the_sun = db.Column(db.Float)
```

Comment:
Consider which, if any, of these columns should be non-nullable. By default, columns are nullable. Should we be able to make a planet with a NULL name, etc?